### PR TITLE
MusicXML: Export color of group-symbol

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
Backport of #20952

### CODE REVIEW

File: importexport/musicxml/exportxml.cpp [hunk 1]
```diff
@@ -4162,10 +4162,15 @@ static void partGroupStart(XmlWriter& xml, int number, BracketType bracket)
...
                   break;
             default:
-                  qDebug("bracket subtype %d not understood", int(bracket));
+                  qDebug("bracket subtype %d not understood", int(bracket->bracketType()));
             }
-      if (br != "")
-            xml.tag("group-symbol", br);
+      if (br != "") {
+            QString tag = "group-symbol";
+            tag += color2xml(toBracket(bracket));
```
[the remark this answers is not in the document]
Comment (Jojo-Schmitz) [reply importexport/musicxml/exportxml.cpp:4169]: @rettinghaus?

[the remark this answers is not in the document]
Comment (rettinghaus) [reply importexport/musicxml/exportxml.cpp:4169]: Why do you want to? Why not hand over the `BracketItem`?

[the remark this answers is not in the document]
Comment (Jojo-Schmitz) [reply importexport/musicxml/exportxml.cpp:4169]: Because in Mu3 that doesn't work

[the remark this answers is not in the document]
Comment (Jojo-Schmitz) [reply importexport/musicxml/exportxml.cpp:4191]: @rettinghaus Mind to have another look at this, at how to fix this issue for Mu3?

[the remark this answers is not in the document]
Comment (Jojo-Schmitz) [reply importexport/musicxml/exportxml.cpp:4191]: @lvinken: maybe you can sched a light on us, or find some altnate way of fixing this issue for 3.x?

[the remark this answers is not in the document]
Comment (lvinken) [reply importexport/musicxml/exportxml.cpp:4191]: Can have a look, but may take a few days.

[the remark this answers is not in the document]
Comment (Jojo-Schmitz) [reply importexport/musicxml/exportxml.cpp:4191]: Eieiei, excellent question: no it does not, so this entire PR is totally mute... (for 3.x that is)

**Pull request closed without merging**